src/DETHRACE/common/displays.h

```diff
@@ -26,7 +26,7 @@ extern int gTimer_headup;
 extern int gTime_awarded_headup;
 extern int gPed_kill_count_headup;
 extern int gDim_amount;
-extern br_pixelmap* gHeadup_images[31];
+extern br_pixelmap* gHeadup_images[32];
```

Comment:
How did you determine this should be 32? `DisposeHeadupImages` is clearly going from index 0 to 30?

Reply:
at the end of the array, at 0x128CA4 I have `gNet_cash_headup`

Reply:
See `LoadHeadupImages` -> `gHeadup_images`.
It's to accommodate the demo timeout fancy head up.

Reply:
This is demo-only behavior.